docs/src/devel/contributing.rst

```diff
@@ -164,6 +164,51 @@ data. A common pattern is validating the data file created by the
 associated export role, reading the data file and then calling the
 module you created for import.
 
+
+Commit's messages
```

Comment:
s/Commit's messages/Commit Messages/

There should not be possessiveness of the messages to the commit. :)  Also, title case.